sdk/communication/azure-communication-chat/src/test/java/com/azure/communication/chat/ChatAsyncClientTest.java

```diff
@@ -48,6 +50,7 @@
     @Override
     protected void beforeTest() {
         super.beforeTest();
+        assumeTrue(Configuration.getGlobalConfiguration().get("TEST_PACKAGES_ENABLED", "").matches(".*(all|chat).*"));
```

Comment:
Do we have a shared class that listed out all the env variable names we used? You can make the ```TEST_PACKAGES_ENABLED``` a constant there and reference it in each test so that if we want to change the param name later, you just need to change the value in one place instead of changing it in every file

Reply:
Yup addressed, thanks Rong!